JLToast.xcodeproj/project.pbxproj

```diff
@@ -450,6 +453,8 @@
 				PRODUCT_BUNDLE_IDENTIFIER = kr.xoul.JLToast;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```

Comment:
[useyourloaf.com/blog/swift-whole-module-optimization/](Here's a write-up on Whole Module Optimization.) Xcode 8 bugs you to turn it on when upgrading the project. For projects that are being built for release (i.e. libraries/frameworks) it is recommend to turn on.